modules/exploits/linux/http/apache_couchdb_cmd_exec.rb

```diff
@@ -63,14 +63,14 @@ def initialize(info = {})
 
   def check
 
-    version = get_version
+    version = Gem::Version.new(get_version)
 
     case
     when !version
       return Exploit::CheckCode::Unknown
-    when version < '1.7.0'
+    when version < Gem::Version.new('1.7.0')
       return Exploit::CheckCode::Appears
-    when version.between?('2.0.0','2.1.0')
+    when version >= Gem::Version.new('2.0.0') && version < Gem::Version.new('2.1.1')
```

Comment:
`Gem::Version` has a `between` method, which is slightly easier to read.

Note that it's version inclusive, so you'll probably want something like this:

```ruby
when version.between?(Gem::Version.new('2.0.0'), Gem::Version.new('2.1.0'))
```